pydantic_ai_slim/pydantic_ai/settings.py

```diff
@@ -88,6 +92,30 @@ class ModelSettings(TypedDict, total=False):
     * Anthropic
     """
 
+    tool_choice: Literal['none', 'required', 'auto'] | list[str] | None
+    """Control which function tools the model can use.
+
+    This setting affects both which tools are sent to the model and the API tool_choice parameter.
```

Comment:
> This setting affects both which tools are sent to the model

It shouldn't unless we absolutely have to, so that should be made clear here (likely in the list of models below, we can explain the circumstances where it's needed to filter)

Reply:
changed the wording